pom.xml

```diff
@@ -0,0 +1,483 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0
+    http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>org.openmicroscopy</groupId>
+  <artifactId>ome-codecs</artifactId>
+  <version>6.0.0-SNAPSHOT</version>
```

Comment:
It predated that discussion.  For ome-jai we picked 0.1.0 because we don't intend to have it reach a 1.0.0 release.  But this component will be immediately used and consumed by bioformats and is already stable; I've set it to 1.0.0-SNAPSHOT for that reason; I can set it to 0.1.0-SNAPSHOT if there's value to make a release before bumping to 1.0.0, but I don't see use for that at present.